src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+    <PropertyGroup>
+        <TargetFramework>$(LatestTargetFramework)</TargetFramework>
+        <!-- Workaround https://github.com/microsoft/MSBuildSdks/issues/469 -->
+        <TargetFrameworks />
+        <IsShipping>false</IsShipping>
+        <!-- Reference the outputs to have them available as build outputs. -->
+        <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+        <IsPackable>true</IsPackable>
+        <IncludeBuildOutput>true</IncludeBuildOutput>
+        <IncludeSymbols>false</IncludeSymbols>
+        <EnablePackageValidation>false</EnablePackageValidation>
+        <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+        <PackageDescription>Internal transport package to provide dotnet-api-docs with the reference assemblies and compiler generated documentation files from dotnet/extensions.</PackageDescription>
+        <NoWarn>$(NoWarn);NU5128;NU5131</NoWarn>
+        <BuildProjectReferences Condition="'$(NoBuild)' == 'true'">false</BuildProjectReferences>
+        <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);IncludeProjectReferenceCompilerGeneratedSecondaryOutputInPackage</TargetsForTfmSpecificContentInPackage>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <!-- Set PrivateAssets to avoid dependencies in the nupkg
```

Comment:
Strictly speaking you don't need PrivateAssets here as you already set `SuppressDependenciesWhenPacking` which emptires the package dependency nodes.